reflex/vars.py

```diff
@@ -1347,12 +1353,14 @@ def __hash__(self) -> int:
         Returns:
             The hash of the var.
         """
-        return hash((self.tag, self.is_default, self.alias))
+        return hash((self.tag, self.is_default, self.alias, self.install, self.render))
 
 
 class NoRenderImportVar(ImportVar):
```

Comment:
Are we deprecating this? Or leaving it as-is?

Reply:
oh, maybe the line moved. I was referring to `NoRenderImportVar`. Are we keeping _that_?

Reply:
Ah yeah, I just kept it in case anyone was using it, but ultimately it will be deprecated/removed yes.
(already removed all reference to it in the codebase)